patientsearch/src/js/components/PatientListTable.js

```diff
@@ -386,7 +386,9 @@ export default function PatientListTable() {
           console.log("Error occurred adding row to table ", e);
         }
         handleRefresh();
-        handleLaunchApp(formatData(response)[0]);
+        // by default launch the first defined client application after account creation
+        // TODO use config variable? to enable / disable certain SoF clients dependent on FHIR state.
```

Comment:
Would this change the behavior of other projects (eg cosri)? 

If yes, we probably want a config (eg `LAUNCH_AFTER_PATIENT_CREATION` or `QUICK_LAUNCH`) to prevent affecting existing systems

Reply:
thank you @ivan-c fixed to use the config variable `LAUNCH_AFTER_PATIENT_CREATION`